crates/bevy_ecs/src/lib.rs

```diff
@@ -82,7 +82,7 @@ pub mod prelude {
 #[cfg(test)]
 mod tests {
```

Comment:
Is this intended as the minimal change to support the new concepts, and tests can be added in a future PR?